src/pages/tutorials/getting-started-with-spring-boot.md

```diff
@@ -160,11 +152,179 @@ This schema defines one top level field (in the type `Query`):  `bookById` which
+    }
+
+    public String getId() {
+        return id;
+    }
+}
+```
+
+## Adding code to fetch data
+Spring for GraphQL provides an [annotation-based programming model](https://docs.spring.io/spring-graphql/docs/current/reference/html/#controllers) to declare handler methods to fetch the data for specific GraphQL fields.
+
+Later in this tutorial we'll discuss how to manually create and register DataFetchers, without using this annotation feature.
+
+Add the following to `book-details/BookController.java`
+
+```java
+@Controller
+public class BookController {
+    @QueryMapping
+    public Book bookById(@Argument String id) {
+        return Book.getById(id);
+    }
+
+    @SchemaMapping
+    public Author author(Book book) {
+        return Author.getById(book.getAuthorId());
+    }
+}
+```
+The `@QueryMapping` annotation binds this method to a query, a field under the Query type.
+The query field is then determined from the method name, `bookById`. It could also be declared on the annotation. 
+Spring for GraphQL uses `RuntimeWiring.Builder` to register the handler method as a `graphql.schema.DataFetcher` for the query field `bookById`.
+
+In GraphQL Java, `DataFetchingEnvironment` provides access to a map of field-specific argument values.
+Use the `@Argument` annotation to have an argument bound to a target object and injected into the handler method.
+By default, the method parameter name is used to look up the argument.
+The argument name can be specified in the annotation.
+
+The `@SchemaMapping` annotation maps a handler method to a field in the GraphQL schema and declares it to be the `DataFetcher` for that field.
+The field name defaults to the method name, and the type name defaults to the simple class name of the source/parent object injected into the method. In this example, the field defaults to `author` and the type defaults to `Book`.
+The type and field can be specified in the annotation.
+
+For more, see the [documentation for the Spring for GraphQL annotated controller feature](https://docs.spring.io/spring-graphql/docs/current/reference/html/#controllers).
+
+That's all the code we need! Let's run our first query.
+
+## Running our first query
+
+### Enable the GraphiQL Playground
+GraphiQL is a useful visual interface for writing and executing queries, and much more. Enable GraphiQL by adding this config to the `application.properties` file.
+
+```
+spring.graphql.graphiql.enabled=true
+spring.graphql.graphiql.path=/graphiql
+```
+
+### Boot the application
+Start your Spring application. 
+
+Navigate to http://localhost:8080/graphiql or your custom URL.
+
+### Run the query
+Type in the query and hit the play button at the top of the window.
+
+```graphql
+query {
+  bookById(id: "book-1") {
+    id
+    name
+    pageCount
+    author {
+      id
+      firstName
+      lastName
+    }
+  }
+}
+```
+
+You should see a response like this.
+![GraphQL response](/img/graphiQL.png)
```

Comment:
I think this image link is wrong - I tried to view it and it didnt show

Reply:
This is a Docusaurus thing - you have to launch the site via `yarn start` to see the images, while in dev mode